service/controller/resource/clusterconfigmap/desired.go

```diff
@@ -15,6 +15,12 @@ import (
 	"github.com/giantswarm/cluster-operator/service/controller/key"
 )
 
+type clusterProfile string
+
+const (
+	xs clusterProfile = "XS"
```

Comment:
As I said [here](https://github.com/giantswarm/nginx-ingress-controller-app/pull/34#pullrequestreview-371062931) I would like more a numeric system (or semver). Comparision would be easier. Not blocking anyway

Reply:
Switched to int based cluster profiles - 0 for unknown, 1 for extra small
@pipo02mix @rossf7 PTAL (here and in nginx app)